### PR TITLE
fix(java): patch java errors and mistakes

### LINKED ISSUE #1841: return ctx.exports();  in java code this method does not exist in Context
### What happened?

https://www.pulumi.com/learn/pulumi-fundamentals/configure-and-provision/ source code shows

image

while return ctx.exports();  does not exist and also  the method returns void, that way return statement is also wrong

image

### Steps to reproduce

look at this page https://www.pulumi.com/learn/pulumi-fundamentals/configure-and-provision/

### Expected Behavior

both api and return statement should be removed

### Actual Behavior

no return statement needed as method returns void

### Output of `pulumi about`

G:\DWork\osource\cloudc\devops\pulumi\java\docker>pulumi about
CLI
Version      3.38.0
Go Version   go1.19
Go Compiler  gc

Plugins
NAME    VERSION
docker  3.2.0
java    unknown

Host
OS       Microsoft Windows 10 Pro
Version  10.0.19045 Build 19045
Arch     x86_64

This project is written in java: executable='d:\Apps\Dev\langs\java\jdk\17.x\bin\java.exe' version='java 17.0.2 2022-01-18 LTS
Java(TM) SE Runtime Environment (build 17.0.2+8-LTS-86)
Java HotSpot(TM) 64-Bit Server VM (build 17.0.2+8-LTS-86, mixed mode, sharing)' java='d:\Apps\Dev\langs\java\jdk\17.x\bin\java.exe' javac='17.0.2'

Backend
Name           pulumi.com
URL            https://app.pulumi.com/nagkumar
User           nagkumar
Organizations  nagkumar

No dependencies found

Pulumi locates its logs in C:\Users\RAJANA~1\AppData\Local\Temp by default
warning: Failed to get information about the current stack: No current stack

G:\DWork\osource\cloudc\devops\pulumi\java\docker>

### Additional context

_No response_

### Contributing

Vote on this issue by adding a 👍 reaction. 
To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already).

### PR DESCRIPTION
**Pull request by nimbinatus**
Updates code to remove old code syntax from before the launch.

fix #1841, fix pulumi/pulumi-java#810

### CODE REVIEW

File: themes/default/content/learn/pulumi-fundamentals/configure-and-provision/index.md [hunk 1]
```diff
@@ -1116,7 +1118,6 @@ public class App {
         );
 
         ctx.export("link", Output.of("http://localhost:3001"));
-        return ctx.exports();
```
Comment (t0yv0) [inline themes/default/content/learn/pulumi-fundamentals/configure-and-provision/index.md:1119]: Good catch 🙇‍♂️

**Review by t0yv0 — APPROVED** [review]
This looks good. Our team wasn't sure about `final` being the proper Java style or not, I can go either way. It's intentionally read-only but whether it's good style to always tell it to the compiler I'm not sure.

**Comment by nimbinatus** [comment]
> This looks good. Our team wasn't sure about `final` being the proper Java style or not, I can go either way. It's intentionally read-only but whether it's good style to always tell it to the compiler I'm not sure.

Ha, I was updating them to match the style we've used elsewhere 😂  Thanks for the review!

**Comment by t0yv0** [comment]
It's good to be consistent then! Sounds like it's our style. Thanks.

**Pull request merged**